content/pages/YakShaver/home.json

```
@@ -140,14 +140,14 @@
       },
       "topRightBox": {
         "title": "Real-Time PBIs",
-        "description": "Stay updated instantly with alerts when PBIs have been created, assigned, or require follow-up"
+        "description": "Stay updated instantly when PBIs have been created, assigned, or require follow-up"
       },
       "counterBox1": {
         "title": "Yaks Shaved",
         "counterMetric": "YakShaved"
       },
       "counterBox2": {
-        "title": "Saving Users",
+        "title": "Users' Time Saved",
```

```suggestion
        "title": "User's Time Saved",
```

I know Tiago said to do this in the video but this is a typo